google/cloud/pubsub/internal/publisher_tracing_connection_test.cc

```diff
@@ -15,35 +15,132 @@
 #ifdef GOOGLE_CLOUD_CPP_HAVE_OPENTELEMETRY
 
 #include "google/cloud/pubsub/internal/publisher_tracing_connection.h"
+#include "google/cloud/pubsub/message.h"
 #include "google/cloud/pubsub/mocks/mock_publisher_connection.h"
 #include "google/cloud/pubsub/publisher_connection.h"
-#include "google/cloud/internal/opentelemetry.h"
-#include "google/cloud/opentelemetry_options.h"
+#include "google/cloud/pubsub/topic.h"
+#include "google/cloud/future_generic.h"
+#include "google/cloud/internal/make_status.h"
+#include "google/cloud/status.h"
+#include "google/cloud/status_or.h"
 #include "google/cloud/testing_util/opentelemetry_matchers.h"
+#include "google/cloud/testing_util/status_matchers.h"
+#include "gtest/gtest.h"
```

Comment:
we should probably remove this include. Or use `<gtest/gtest.h>` if it is actually necessary.

Reply:
Yeah.... I shouldn't accept clang-tidys help.

Whoops.

Removed.